test/manipulation.test.js

```diff
@@ -743,7 +743,15 @@ describe('manipulation', function() {
       Todo = db.define('Todo', {
         content: String,
       });
-      db.automigrate(['Post', 'Todo'], done);
+      Person = db.define('Person', {
+        name: String,
+        gender: String,
+        married: Boolean,
+        age: {type: Number, index: true},
+        dob: Date,
+        createdAt: {type: Date, default: Date},
```

Comment:
Makes sense. Unfortunately, this is not clear from the code. Can you please add a comment explaining why the Person model is changed for these tests?

It would be great to eventually clean up these tests so that we don't turn on/turn off `forceId` flag in a seemingly random way...